src/styles/antisnake.scss

```diff
@@ -0,0 +1,44 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(3, 3fr);
+
+  &__block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-width: 300px;
+    min-height: 300px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+    color: white;
+  }
+  
+  $color-list: 
+    "1" rgb(255, 0, 0),
+    "2" rgb(204, 0, 0),
+    "3" rgb(153, 0, 0),
+    "4" rgb(102, 0, 0),
+    "5" rgb(51, 0, 0),
+    "6" rgb(0, 0, 0);
+  @each $number, $value in $color-list {
+    &__block:nth-child(n + #{$number})::before {
+      content: $number;
+    }
+
+    &__block:nth-child(n + #{$number}) {
+      background-color: $value;
+    }
```

Comment:
Use `&` and avoid such nesting. 
Move this loop to `&__block` selector

Reply:
+